.github/workflows/ci.yml

```diff
@@ -134,6 +134,10 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
         with:
             toolchain: ${{ env.rust_stable }}
+      - name: Install Rust ${{ env.rust_nightly }}
+        uses: dtolnay/rust-toolchain@stable
+        with:
+            toolchain: ${{ env.rust_nightly }}
```

Comment:
Please don't install multiple Rust versions in a single CI run. It causes confusion about which version of Rust is actually being used in each call. Instead, create a new CI run for this check.

Reply:
Okay. Created a new CI run.